packages/components/src/Slider/Slider.component.js

```diff
@@ -239,20 +239,23 @@ class Slider extends React.Component {
 			...rest
 		} = this.props;
 		const noValue = value === null || value === undefined;
+		const Component = Array.isArray(value) ? Range : RcSlider;
 		return (
 			<div>
 				<div className={classnames(theme['tc-slider'], 'tc-slider')}>
-					<RcSlider
-						id={id}
-						value={value}
-						min={min}
-						max={max}
-						handle={noValue ? undefined : this.state.handle}
-						className={classnames(theme['tc-slider-rc-slider'], 'tc-slider-rc-slider')}
-						onChange={onChange}
-						disabled={disabled}
-						{...rest}
-					/>
+					{
```

Comment:
why is there this bracket around the Component  ?

Reply:
good catch, I fix it